lib/gems/pending/util/extensions/miq-benchmark.rb

```diff
@@ -29,6 +29,10 @@ def self.realtime_block(key, &block)
     begin
       ret = realtime_store(hash, key, &block)
       return ret, hash
+    rescue
+      # Don't let timings be lost on exception when there is nobody else to pick them up.
+      $log.info("Exception in realtime_block #{key.inspect} - Timings: #{hash.inspect}") if outermost
```

Comment:
> Try that first and see if the workers log it.

Again, you have to put on your non-manageiq hat, because that's where the class is going.  This makes no sense if some random project uses it.